postmark/core.py

```diff
@@ -402,11 +419,14 @@ def to_json_message(self):
 
         return json_message
 
-    def send(self, test=None):
+    def send(self, endpoint=None, test=None):
         '''
         Send the email through the Postmark system.
         Pass test=True to just print out the resulting
         JSON message being sent to Postmark
+
+        endpoint: use this to post to other Postmark API endpoints. If not set, the default is "email"
+
```

Comment:
Where is this default `email` coming in? Shouldn't it be in the function declaration?